thirdparty/openapi3/schemas/ConsentRequestsIDPutResponseOTPAuth.yaml

```diff
@@ -0,0 +1,43 @@
+title: ConsentRequestsIDPutResponseOTPAuth
+type: object
+description: |
+  The object sent in a `PUT /consentRequests/{ID}` request.
+
+  Schema used in the authentication phase of the account linking flow,
+  the user is expected to prove their identity to the DFSP by passing a OTP
+  or secret to the PISP.
+properties:
+  id:
+    $ref: '../../../v1.0/openapi3/schemas/CorrelationId.yaml'
+  initiatorId:
+    type: string
+    description: The id of the PISP who will initiate transactions on a user's behalf.
+  scopes:
+    type: array
+    items:
+      $ref: './Scope.yaml'
+  authChannels:
```

Comment:
Can we enforce that this is exactly an array containing `["OTP"]`?

Reply:
I think so...I think I can do an array with enums.

created array enums that only contain `["OTP"]` and another containing `["Web"]`